2024-MacC-M4-6princess/2024-MacC-M4-6princess/Source/View/Camera/CamTakepicCountView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CameraTakepicCountView.swift
+//  2024-MacC-M4-6princess
+//
+//  Created by 김이예은 on 10/7/24.
+//
+
+import SwiftUI
+
+//sprint2에 사용합니다
+struct CameraTakepicCountView: View {
```

Comment:
P3: 코드 중복 최소화

```suggestion
struct CameraTakepicCountView: View {
    @Binding var takePicCount: Int
    
    let takePicOptions = [1, 2, 4, 8]
    
    var body: some View {
        VStack(alignment: .center, spacing: 10) {
            ForEach(takePicOptions, id: \.self) { count in
                Button {
                    self.takePicCount = count
                    print("지금부터 \(count)컷의 사진이 연속으로 촬영됩니다.")
                } label: {
                    Text("\(count)번 촬영")
                }
            }
        }
    }
}
```

ForEach로 간결하게 정리할 수 있을 것 같아요 : )

Reply:
감사합니다!